src/types.ts

```diff
@@ -56,6 +56,16 @@ export type AutomoxDevice = {
     VENDOR: string;
     VERSION: string;
   };
+  status?: {
+    device_status?: string;
+    agent_status?: string;
+    policy_status?: string;
+    policy_statuses?: { id: number }[];
```

Comment:
What is the difference between these two fields? I'm surprised to see `policyStatus` as a string, and `policyStatuses` as numbers.

Reply:
Oh, I see. Should we type like this then?
```suggestion
    policy_status?: string;
    policy_statuses?: { id: number, compliant: boolean }[];
```

Reply:
sure!